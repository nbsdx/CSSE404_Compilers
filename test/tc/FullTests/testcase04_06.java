class Main {
	public static void main(String[] args) {
	}
}

class Foo {
	boolean bool;
	public int meth(boolean ignore) {
		return 5;
	}
}

class Bar extends Foo {
	public int meth2(boolean abc) {
		return this.meth(3);
	}

}
