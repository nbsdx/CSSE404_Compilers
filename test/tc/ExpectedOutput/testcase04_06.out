Argument type int is incompatible with formal parameter type boolean
